src/sg/gov/acra/bizfiles/profiles/business/1.0/business-profile-open-attestation-sample-document.json

```diff
@@ -0,0 +1,376 @@
+{
+	"entityName": "Eat toast bread",
+	"formerNames": [
+		"#!T*-#&T?!ST"
+	],
+	"changeOfNameDate": "2019-10-18",
+	"uen": "53235918J",
+	"registrationDate": "2019-07-31",
```

Comment:
all the profiles template uses incorporationDate as field label. should we follow the same ?